src/main/java/net/openhft/chronicle/bytes/BytesStore.java

```diff
@@ -353,18 +353,13 @@ default boolean inside(@NonNegative long offset) {
 
     /**
      * Returns if a number of bytes starting from an offset are inside this ByteStore limits.
-     * If you are going to write n bytes starting at offset, you need to call this method with buffer=n-1,
-     * as the 1st byte is written at offset, and the last at offset+n-1
-     * <p>
-     * Use this test to determine if an offset is considered safe to write to. Note that it checks we are
-     * inside the BytesStore limits *including* the overlap
```

Comment:
I still feel strongly that we make this intuitive rather than writing a convoluted comment as to why it's not and how to use it correctly.